File: wowsinfo/ios/wowsinfo/usecase/react-native/communication/NativeHandler.m
//
//  NativeHandler.m
//  wowsinfo
//
//  Created by Yiheng Quan on 27/8/2023.
//

#import <React/RCTBridgeModule.h>

@interface RCT_EXTERN_MODULE(NativeHandler, NSObject)

RCT_EXTERN_METHOD(saveMainAccountInfo:(NSString *)json)

@end

File: apps/mobile/ios/wowsinfo/usecase/react-native/communication/NativeHandler.swift
//
//  NativeHandler.swift
//  wowsinfo
//
//  Created by Yiheng Quan on 27/8/2023.
//

import React

/// Handles all not UI related requests from React Native
@objc(NativeHandler)
class NativeHandler: NSObject, RCTBridgeModule {
    
    @objc func saveMainAccountInfo(_ json: String) {
        // TODO: to be implemented
    }
    
    static func moduleName() -> String! {
        "NativeHandler"
    }
    
    static func requiresMainQueueSetup() -> Bool {
        false
    }
}
